Makefile

```diff
@@ -4,11 +4,18 @@ NAME := logging-app
 OPS_URL ?= https://opscenter.localhost.localdomain:33009
 OUT ?= $(NAME).tar.gz
 GRAVITY ?= gravity
+LOGRANGE_VERSION := 0.1.46
```

Comment:
Shouldn't it be `v0.1.46`? Or is the image published both with and without the "v" prefix?

Reply:
fixed.

Reply:
@eldios Going forward, please make sure to test the changes and verify they work before opening a pull request. This issue could have been very easily caught beforehand since there's no such image and it just wouldn't work:

```
➜  ~ docker pull index.docker.io/logrange/logrange:0.1.46
Error response from daemon: manifest for logrange/logrange:0.1.46 not found: manifest unknown: manifest unknown
```

Reply:
@r0mant I tested the new logrange version with the hard-coded version and then remembered your suggestion to introduce the "set-image" approach. Thus added that too as a last thing after I opened the PR.

Reply:
Yep, no problem. Shows why it's a good idea to retest after refactoring as well :)